### DISCUSSION TITLE
Question about Holley DTZ541

### DISCUSSION THREAD

**Discussion by dl1hrc** [arendst/Tasmota#18974] [Q&A]
Hi,
I use the tasmota sendor and get back the the total power consumption as described in the documentation:
_Make sure to enable info in the settings, otherwise you only get total / current consumption and total export._
My question is what settings I have to change? Something in the tasmota-sw or in the smart meter?
I thank you very much for a short hint.
Adi

**Comment by sfromis** [comment]
As I read it, it is settings done in your smartmeter, and I'd expect the manual to include such details. Maybe only available online....

**Comment by dl1hrc** [comment]
Ok, thank you. Yes, i think so too. I asked the seller, we'll see...

**Reply by sfromis** [reply]
I'd first search the net for a manual. It is very common for a PDF to exist for such info.
In fact, results of Googling for *DTZ541 filetype:pdf* looks very relevant.
